doc/user-guide.md

```diff
@@ -1,73 +1,72 @@
 # User Guide
 
-***Note:*** This guide describes how to use the new *ordered* option in Gorums.
-This option will likely become the default at some point in the future.
-
 You may wish to read the gRPC "Getting Started" documentation found [here](http://www.grpc.io/docs/) before continuing.
 Gorums uses gRPC under the hood, and exposes some of its configuration.
+Gorums also uses [Protocol Buffers](https://developers.google.com/protocol-buffers) to specify messages and RPC methods.
 
 ## Prerequisites
 
 This guide describes how to use Gorums as a user.
 The guide requires a working Go installation.
-At least Go version 1.13 is required.
+At least Go version 1.16 is required.
 
 There are a few tools that need to be installed first:
 
-First, we need version 3 of `protoc`, the Protocol Buffers Compiler.
-Installation of this tool is OS/distribution specific.
-See <https://github.com/google/protobuf/releases> and <https://developers.google.com/protocol-buffers> for details.
+1. Install version 3 of `protoc`, the Protocol Buffers Compiler.
+   Installation of this tool is OS/distribution specific.
 
-Second, we need to install the [Go code generator](https://github.com/protocolbuffers/protobuf-go) for `protoc`.
-It can be installed with the following command:
+   On macOS with Homebrew you can use:
```

Comment:
Note: This works on Linux too

Reply:
Using homebrew? So I can just say On Linux/macOS with Homebrew?

Reply:
Yes. Homebrew can be used on Linux. It may be preferable because the `protoc` version included in a distro's package manager might be outdated, and it is easier than installing it from a tarball.